src/components/views/spaces/SpaceCreateMenu.tsx

```diff
@@ -88,7 +88,13 @@ const SpaceCreateMenuType: React.FC<{
     onClick(): void;
 }> = ({ title, description, className, onClick }) => {
     return (
-        <AccessibleButton className={classNames("mx_SpaceCreateMenuType", className)} onClick={onClick}>
+        <AccessibleButton
+            className={classNames("mx_SpaceCreateMenuType", className)}
+            onClick={onClick}
+            aria-label={_t("Create a %(publicOrPrivate)s space", {
```

Comment:
Why is the aria label different from the button text contents? This button already has a good accessible name. Our testing utils should be flexible enough to select buttons in multiple ways as needed.

Reply:
This is based on the same thought written below: https://github.com/matrix-org/matrix-react-sdk/pull/10620/files#r1168337926.

I was wondering if actual a11y users would get the strings which consist of both `{title}` and `{description}` as the button's label text.

If using slashes for selecting the button's text in a test like `cy.findByRole("button", { name: /title/ })` is fine and expected to emulate a11y tool experience, this change is not needed.